draft-ietf-quic-transport.md

```diff
@@ -2735,6 +2742,22 @@ also be forward-secure encrypted.  Since the attacker will not have the forward
 secure key, the attacker will not be able to generate forward-secure encrypted
 packets with ACK frames.
 
+## Slowloris Attacks
+
+The attacks commonly known as Slowloris {{SLOWLORIS}}
+try to keep many connections
+to the target endpoint open and hold them open as long as possible. These attacks
+can be executed against a QUIC endpoint by slowly sending small amount of
+data on an open stream, slowly opening the flow control windows in order
+to limit the sender rate, or possibly manufacturing QUIC ACK that
+simulate a high loss rate and slow down sending by the other endpoint.
+
+QUIC implementers should consider implementing the
+classic mitigations of the Slowloris attack, such as increasing the maximum
```

Comment:
"the classic" isn't needed :)  How about "QUIC deployments SHOULD provide mitigations for the Slowloris attack, such as ..."